### PR TITLE
Detailed maps

### PR DESCRIPTION
**Pull request by francistse23**
Addresses issue #97 

- Added detailed version of maps for The Skeld, Mira HQ, and Polus
- Added Reddit users to contributors in `readme` for allowing us to use the maps

As usual, this version is pushed to my github pages. Dont know how I feel about the text colors to be honest... 

Hard to find a color that works well with different background colors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Kermitigated (twitch handle)
 [Victor Pietro](https://github.com/pietroow) (A.K.A. Pietroow)  
 [Coleyra](https://github.com/Coleyra)  
 [MissingNoShiny](https://github.com/MissingNoShiny)  
+u/Stark_001001 (The Skeld in game map)  
+u/mooseknuckle_king (Mira HQ and Polus in game maps)
```
Comment (Kedyn) [inline README.md:55]: Maybe we should add  their names and link them to their reddit profile.

**Review by Kedyn — COMMENTED** [review]
I do not think we need to have a feature to switch between them, replacing the maps is just fine. Plus although the author's twitter handle for the original maps is on it, I did not really get permission so I think just replacing them would be best.

File: README.md (hunk 1 above)
Comment (francistse23) [reply README.md:55]: didn't really get their names.. just going off their username on Reddit. I can link their profiles though

Comment (francistse23) [reply README.md:55]: do you find the text hard to read?

Comment (Kedyn) [reply README.md:55]: Yes, the color is a bit off.

**Review by Kedyn — APPROVED** [review]

**Pull request merged**